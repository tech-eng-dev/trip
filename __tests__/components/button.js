import React from 'react';
import renderer from 'react-test-renderer';
import CustomButton from '../../src/components/button';


test('CustomButton renders correctly', () => {
  const button = renderer.create(<CustomButton
    text="test"
    testID="test"
    onPress={() => {}}
    backgroundColor="black"
  />).toJSON();
  expect(button).toMatchSnapshot();
});
