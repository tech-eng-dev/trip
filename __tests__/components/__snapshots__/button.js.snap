// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomButton renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  collapsable={undefined}
  hasTVPreferredFocus={undefined}
  hitSlop={undefined}
  isTVSelectable={true}
  nativeID={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "black",
      "borderRadius": 4,
      "height": 40,
      "justifyContent": "center",
      "marginVertical": 10,
      "opacity": 1,
      "width": 500,
    }
  }
  testID="test"
  tvParallaxProperties={undefined}
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#F5FAFE",
        "fontSize": 18,
        "textAlign": "center",
      }
    }
  >
    test
  </Text>
</View>
`;
